src/phantasm-renderer/backend/detail/linked_pool.hh

```diff
@@ -13,25 +15,39 @@ struct linked_pool
     static_assert(sizeof(T) >= sizeof(T*), "linked_pool element type must be large enough to accomodate a pointer");
     using index_t = IdxT;
 
+    linked_pool() = default;
+    linked_pool(linked_pool const&) = delete;
+    linked_pool(linked_pool&&) noexcept = delete;
+    linked_pool& operator=(linked_pool const&) = delete;
+    linked_pool& operator=(linked_pool&&) = delete;

```

Comment:
sound like a good data structure for cc, right?

Reply:
Yeah, there are now a number of special purpose containers in backend/detail which could be (in part) moved to cc eventually, other prime candidates are `unique_buffer` and `cache_map`